libats-http/src/main/scala/com/advancedtelematic/libats/http/ValidatedGenericMarshalling.scala

```diff
@@ -0,0 +1,15 @@
+package com.advancedtelematic.libats.http
+
+import akka.http.scaladsl.unmarshalling.{FromStringUnmarshaller, Unmarshaller}
+import akka.http.scaladsl.util.FastFuture
+import com.advancedtelematic.libats.data.ValidatedGeneric
+
+object ValidatedGenericMarshalling {
```

Comment:
Why do se need the marshaller? It should be possible to derive it from circe codec